ext/RMagick/rmimage.c

```diff
@@ -9061,6 +9061,9 @@ Image_monitor_eq(VALUE self, VALUE monitor)
         (void) SetImageProgressMonitor(image, rm_progress_monitor, (void *)monitor);
     }
 
+#if defined(_WIN32)
```

Comment:
Is this only a problem on ImageMagick 6.8, or does it also occur on later versions? Wondering if we can add a check for the version.

Reply:
Sorry I don't investigate it. However, I think it is tough to check ImageMagick version at runtime in here.

Reply:
Okay, how about we add a comment then? I just want to make sure we can track this down and remove it when we drop 6.8 support.

```c
// broken on 6.8
```